e2e-tests/e2e-fixtures/dateConstants.js

```diff
@@ -134,6 +134,16 @@ const sixYearsOneDayDay = format(sixYearsOneDay, longDayFormat);
 const sixYearsOneDayMonth = format(sixYearsOneDay, longMonthFormat);
 const sixYearsOneDayYear = format(sixYearsOneDay, longYearFormat);
 
+const twelveMonthsOneDay = add(today, { months: 12, days: 1 });
+const twelveMonthsOneDayDay = format(twelveMonthsOneDay, longDayFormat);
+const twelveMonthsOneDayMonth = format(twelveMonthsOneDay, longMonthFormat);
+const twelveMonthsOneDayYear = format(twelveMonthsOneDay, longYearFormat);
+
+const twelveMonthsOneDayAgo = sub(today, { months: 12, days: 1 });
+const twelveMonthsOneDayAgoDay = format(twelveMonthsOneDayAgo, longDayFormat);
+const twelveMonthsOneDayAgoMonth = format(twelveMonthsOneDayAgo, longMonthFormat);
+const twelveMonthsOneDayAgoYear = format(twelveMonthsOneDayAgo, longYearFormat);
```

Comment:
I have a WIP PR to improve these date values - so for now I'm adding them in the existing format consistent with the other values in this file, but I'm going to prepose a refactor to this soon.

Reply:
@BethThomas141 Sounds good - can we touch base first though please, I have a WIP PR that improves consumption of the date constants https://github.com/UK-Export-Finance/dtfs2/pull/3556